### DISCUSSION TITLE
[NMS] Failed stopping call trace: Internal Server Error

### DISCUSSION THREAD

**Discussion by haidarx** [magma/magma#12547] [Q&A]
Hello,
I was trying to perform call tracing during our testing, and I think the connection from Orc8r to the AGW got disconnected during tracing. 
Then I tried to stop the trace using Actions > Stop and received Failed stopping call trace: Internal Server Error.
If I try to download, it says the trace is still active. 

![Failed stopping call trace - Internal Server Error](https://user-images.githubusercontent.com/24651795/164883073-fa0e8582-9cdc-4405-b293-a770b288d8ed.png)

Is there a way to remove this? I have access to the Kubectl.

Your help is greatly appreciated. Thank you.

BR,
haidar

**Comment by asantos-vk** [comment]
Same
